examples/HelloWorldExample/CMakeLists.txt

```diff
@@ -4,10 +4,6 @@
 
 cmake_minimum_required(VERSION 3.13)
 
-if(NOT CMAKE_VERSION VERSION_LESS 3.0)
-    cmake_policy(SET CMP0048 NEW)
-endif()
-
```

Comment:
This is an autogenerated file coming from Fast DDS Gen when the `-python` option is enabled. I suggest applying these changes directly to the generation tool instead of changing the generated file each time the types has to be regenerated.

Reply:
See eProsima/Fast-DDS-Gen#86